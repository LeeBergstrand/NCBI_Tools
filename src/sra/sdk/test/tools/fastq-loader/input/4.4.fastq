@BILLIEHOLIDAY_1_FC20F3DAAXX:8:2:342:540/1
G12312312312312312312312312312312312
@BILLIEHOLIDAY_1_FC21F3DAAXX:8:2:342:540/1
G12312312312312312312312312312312312
@BILLIEHOLIDAY_1_FC22F3DAAXX:8:2:342:540/1
G12312312312312312312312312312312312
@BILLIEHOLIDAY_1_FC23F3DAAXX:8:2:342:540/1
G12312312312312312312312312312312312
@BILLIEHOLIDAY_1_FC24F3DAAXX:8:2:342:540/1
G12312312312312312312312312312312312
@BILLIEHOLIDAY_1_FC25F3DAAXX:8:2:342:540/1
G12312312312312312312312312312312312

