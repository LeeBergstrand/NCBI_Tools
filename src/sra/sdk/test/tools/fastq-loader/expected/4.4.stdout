     ALIGNMENT_COUNT: 0, 0
          BASE_COUNT: 105
      BIO_BASE_COUNT: 105
      CMP_BASE_COUNT: 105
          CMP_CSREAD: 12312312312312312312312312312312312
        COLOR_MATRIX: 0, 1, 2, 3, 4, 1, 0, 3, 2, 4, 2, 3, 0, 1, 4, 3, 2, 1, 0, 4, 4, 4, 4, 4, 4
              CS_KEY: GT
         MAX_SPOT_ID: 3
         MIN_SPOT_ID: 1
                NAME: 1
            PLATFORM: SRA_PLATFORM_UNDEFINED
PRIMARY_ALIGNMENT_ID: 0, 0
             QUALITY: 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0
         READ_FILTER: SRA_READ_FILTER_PASS, SRA_READ_FILTER_PASS
            READ_LEN: 35, 0
          READ_START: 0, 35
           READ_TYPE: SRA_READ_TYPE_BIOLOGICAL, SRA_READ_TYPE_TECHNICAL
          SPOT_COUNT: 3
          SPOT_GROUP: 
             SPOT_ID: 1
          TRIM_START: 0

     ALIGNMENT_COUNT: 0, 0
          BASE_COUNT: 105
      BIO_BASE_COUNT: 105
      CMP_BASE_COUNT: 105
          CMP_CSREAD: 12312312312312312312312312312312312
        COLOR_MATRIX: 0, 1, 2, 3, 4, 1, 0, 3, 2, 4, 2, 3, 0, 1, 4, 3, 2, 1, 0, 4, 4, 4, 4, 4, 4
              CS_KEY: GT
         MAX_SPOT_ID: 3
         MIN_SPOT_ID: 1
                NAME: 2
            PLATFORM: SRA_PLATFORM_UNDEFINED
PRIMARY_ALIGNMENT_ID: 0, 0
             QUALITY: 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0
         READ_FILTER: SRA_READ_FILTER_PASS, SRA_READ_FILTER_PASS
            READ_LEN: 35, 0
          READ_START: 0, 35
           READ_TYPE: SRA_READ_TYPE_BIOLOGICAL, SRA_READ_TYPE_TECHNICAL
          SPOT_COUNT: 3
          SPOT_GROUP: 
             SPOT_ID: 2
          TRIM_START: 0

     ALIGNMENT_COUNT: 0, 0
          BASE_COUNT: 105
      BIO_BASE_COUNT: 105
      CMP_BASE_COUNT: 105
          CMP_CSREAD: 12312312312312312312312312312312312
        COLOR_MATRIX: 0, 1, 2, 3, 4, 1, 0, 3, 2, 4, 2, 3, 0, 1, 4, 3, 2, 1, 0, 4, 4, 4, 4, 4, 4
              CS_KEY: GT
         MAX_SPOT_ID: 3
         MIN_SPOT_ID: 1
                NAME: 3
            PLATFORM: SRA_PLATFORM_UNDEFINED
PRIMARY_ALIGNMENT_ID: 0, 0
             QUALITY: 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0
         READ_FILTER: SRA_READ_FILTER_PASS, SRA_READ_FILTER_PASS
            READ_LEN: 35, 0
          READ_START: 0, 35
           READ_TYPE: SRA_READ_TYPE_BIOLOGICAL, SRA_READ_TYPE_TECHNICAL
          SPOT_COUNT: 3
          SPOT_GROUP: 
             SPOT_ID: 3
          TRIM_START: 0

